fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/indexes/TextIndexTest.java

```diff
@@ -2349,7 +2349,7 @@ void queryComplexDocumentsWithAdditionalFilters() throws Exception {
                     queryComplexDocumentsWithOr((OrComponent) Query.or(
                             Query.field("text").text().containsAll("unit named after"),
                             Query.field("text").text().containsPhrase("אן ארמיי און פלאט")
-                    ), 0, -1558384887));
+                    ), 0, -396375489));
```

Comment:
This changed because the `comparisonKey` was adjusted from `concatenateFields("group", "rec_no")` (i.e., the primary key) to just `field("rec_no")`, as the `group` field was in the set of equality-bound predicates. I could see an argument that we should include the primary key back in here, I guess, though it's not strictly necessary, but it may decrease the number of changed plans

Reply:
I would imagine this lines up with the plan given by the Cascades planner which IIRC returns the shortest comparison key.